lbr_examples/launch/lbr_ext_torque_example.launch.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
this launch file adds redundancy to the `lbr_bringup` package, should be replaced in favor for `lbr_bringup`, might require update of `lbr_bringup`